public/auth.js

```diff
@@ -2,7 +2,7 @@ class AuthController {
     // Authentication state listener
     static getUser() {
         firebase.auth().onAuthStateChanged(user => {
-            if (user) {
+            if (user && user.email.match(".*@africasvoices.org$")){
```

Comment:
Yeah I get it. The problem is that there are firestore rules that specify who can read from the database. There error `permission denied` comes when a user not using avf email logs in and he/she is denied to read from the db because of the rule below; for one to read the db the email should be avf's. That's why I have implemented the same on the app, since there is no reason to enable a person to login and cannot access data.
```
rules_version = "2";
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read: if request.auth.token.email_verified && request.auth.token.email.matches(".*@africasvoices.org$")
    }
  }
}
```

Reply:
Please try to login with approved gmail address and you will see the issue above

Reply:
Am aware of the above rules. We have external collaborators who access the ops-dashboard using non-avf email address. Adding this check-in frontend locks them out from log-in whereas their authentication is handled in firebase. What we simply need here is a friendly frontend alert to users with details as to why their auth was declined.

Reply:
Updated authentication to give alert on denied permission